### ISSUE TITLE
potential false positive "people people" in lexical illusions

### ISSUE THREAD

**Issue by Nytelife26** [amperser/proselint#1410]
> Maybe we should invert the lexical illusions so that only the words that, when doubled, make sense in some situations. I think this because pretty much any word doubled doesn't make sense:
> [...]
> people people

 _Originally posted by @alexyorke in [#549](https://github.com/amperser/proselint/issues/549#issuecomment-235264765)_

Interestingly, "people people" (along the same lines as "cat people") would be a valid form. I was quite surprised to check [Ngram Viewer](https://books.google.com/ngrams/graph?content=cat+person%2Cpeople+person%2Ccat+people%2Cpeople+people&year_start=1800&year_end=2022&corpus=en&smoothing=3) and find that "people people" is almost as common as "cat people."

**Comment by suchow** [comment]
People hurt people.
Hurt people hurt people people.
People people hurt people hurt hurt people people.

[[[People people] [hurt people] hurt] hurt [people people]]

**Comment by Nytelife26** [comment]
Interesting examples! The mere act of trying to read that last one for too long made "hurt" look less like an actual word.

The aim of the lexical illusions check is clarity and intention. Aside from "hurt people hurt people" being cliché, the first two are reasonably clear, and the third example has bigger issues of clarity to address than "people people."

Here's the current output from running against those examples.

```
<stdin>:2:1: lexical_illusions There's a lexical illusion in 'Hurt people hurt people' - a phrase is repeated.
<stdin>:3:1: lexical_illusions There's a lexical illusion in 'People people' - a phrase is repeated.
<stdin>:3:27: lexical_illusions There's a lexical illusion in 'hurt hurt' - a phrase is repeated.
<stdin>:3:37: lexical_illusions There's a lexical illusion in 'people people' - a phrase is repeated.
```

Taking the "people people" results out would reduce noise in this case, without sacrificing any useful information.
